modules/virt-removing-interface-from-nodes.adoc

```diff
@@ -5,8 +5,10 @@
 [id="virt-removing-interface-from-nodes_{context}"]
 = Removing an interface from nodes
 
-Remove an interface from nodes by editing the `NodeNetworkConfigurationPolicy` object and set
-the `state` of the interface to `absent`.
+Remove an interface from nodes by editing the `NodeNetworkConfigurationPolicy` object and set the `state` of the interface to `absent`.
+
+Removing an interface from a node does not automatically restore the node network configuration to a previous state. After you remove an interface, any of the node NICs throughout the cluster that were previously attached or subordinate to the interface are placed in a `down` state and become unreachable. +
+To maintain connectivity with the NIC, specify an active NIC configuration in the same policy.
```

Comment:
What does `specify an active NIC configuration` mean? Is this a way of saying to enable dhcp?

Reply:
Intended as with a status of 'up'. I have now updated this to be more clear:
"To maintain connectivity with the NIC, specify a NIC configuration in the same policy with a status of `up` and either DHCP or a static IP address."
and in the procedure:
"The following example removes a Linux bridge. The policy also configures the `eth1` NIC with DHCP to avoid losing connectivity:"

Reply:
Oh, I see. Thanks, it's crystal clear now.
/lgtm